README.md

```diff
@@ -1,5 +1,10 @@
 # bookend
 
+<p align="center">
+  <a href="https://www.youtube.com/watch?v=P1Sq7T9PvP0">
+      <img width="460" height="300" src="figures/video_screenshot.png">
+  </a>
+</p>
 bookend is a classifier for text data. We have built a machine learning model to identify the author of a work given only a selection of text. We are continually adding to the model and improving it, so make sure to check back for updates!
 
```

Comment:
Add one more new line here, maybe! For some reason, there is no gap between the two paragraphs in your branch.

Reply:
done